eva-accession-import-dbsnp2/src/main/resources/application.properties

```diff
@@ -1,9 +1,10 @@
 spring.batch.job.names=IMPORT_DBSNP_JSON_VARIANTS_JOB
 
 # DbSNP JSON input source
-parameters.input=src/test/resources/input-files/test-dbsnp.json.bz2
+parameters.input=src/test/resources/input-files/json/test-dbsnp.json.bz2
```

Comment:
Properties files in `src/main/resources` should not be populated with test values.